billing.py

```diff
@@ -30,24 +22,28 @@
 es_service = os.environ.get("ES_SERVICE", "localhost")
 es = Elasticsearch(['http://' + es_service + ':9200/'])
 
+
 @login_manager.user_loader
 def load_user(user_id):
     return User.query.get(int(user_id))
 
```

Comment:
why using 3 lines between def's? is because of the decorator?

Reply:
Yes. PEP8 will complain if there are no 2 blank lines between defs. If there is a decorator, the decorator needs to be right above the def, but two lines of space between the previous def